plasmapy/utils/decorators/converter.py

```diff
@@ -0,0 +1,38 @@
+"""
+Decorator to convert units of functions in /physics methods
+"""
+from astropy import units as u
+import functools
+import inspect
+from plasmapy.utils.decorators import preserve_signature
+__all__ = ["from_radians_to_hz"]
```

Comment:
PEP8 standards have the `__all__` above the imports

Reply:
They do? Cool, TIL, thanks!

Reply:
Yeah, this rule applies to any module level dunder.  The only import statement that precedes dunders is `from __future__ ...`,

https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

Reply:
changing it on the next push